apps/src/lab2/views/Lab2ShareDialogWrapper.tsx

```diff
@@ -79,6 +79,12 @@ const Lab2ShareDialogWrapper: React.FunctionComponent<
 
   const onGoBack = () => {
     setDialogPanel('share');
+    // If the project was submitted successfully, the submission status is updated.
+    if (channelId && projectType) {
+      getSubmissionStatus(channelId, projectType).then(response =>
+        setSubmissionStatus(response?.status)
```

Comment:
Doesn't `getSubmissionStatus` already return `response.status`? Do we need to unwrap it again here?

Reply:
`getSubmissionStatus` currently returns `response.value` which is a `SubmissionStatusResponse` type.
https://github.com/code-dot-org/code-dot-org/blob/5c8feaea12b2e1fc25693db8d6ebaa0490dd6aef/apps/src/templates/projects/submitProjectDialog/submitProjectApi.ts#L6-L8
But I could just set it to return `SubmissStatusType` and unwrap in `getSubmissionStatus`.  I'll update.